README.md

```diff
@@ -215,12 +248,7 @@ gmusic.on('change:song', function (song) {
 });
 ```
 
-- song `Object` - Container for song info
-    - title `String` - Name of the song
-    - artist `String` - Artist of the song
-    - album `String` - Album of the song
-    - art `String` - URL for album art of the song
-    - duration `Number` - Milliseconds that the track will last for
+See `playback.getSongInfo()` for return value.
```

Comment:
Ahhh, actually we should say that `song` will be same as return value of `playback.getSongInfo()`. We aren't "returning" anything =/

```
- song `Object`- Same as return value of `playback.getSongInfo()`
```